### PR TITLE
[AA-docs] update aa-sdk docs to use entryPointWrapper

### PR DESCRIPTION
**Pull request by souradeep-das**
:clipboard: associated with #784 

## Overview

update aa-sdk docs to use entryPointWrapper instead of senderCreatorAddress

## Changes

/

## Testing

/

### CODE REVIEW

File: boba_documentation/developer/account-abstraction/bundler-sdk.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ async function wrapProvider(
   config: ClientConfig,
   originalSigner: Signer = originalProvider.getSigner(), // @ethersproject/abstract-signer
   wallet?: Wallet, // ethers, must be passed
-  senderCreatorAddress?: string // must be passed
+  entryPointWrapperAddress?: string // must be passed
```
Comment (wsdt) [inline boba_documentation/developer/account-abstraction/bundler-sdk.md]: if it's mandatory, why don't we make it a mandatory function argument? 
Or is this code used somewhere else too where we don't want to break things?

**Review by wsdt — APPROVED** [review]
looks good! Left a comment

File: boba_documentation/developer/account-abstraction/bundler-sdk.md (hunk 1 above)
Comment (InoMurko) [reply boba_documentation/developer/account-abstraction/bundler-sdk.md]: Tests would need to be adjusted. But doable

**Review by InoMurko — APPROVED** [review]

File: boba_documentation/developer/account-abstraction/bundler-sdk.md (hunk 1 above)
Comment (souradeep-das) [reply boba_documentation/developer/account-abstraction/bundler-sdk.md]: its optional for the low-level API, but mandatory for the high-level API, updated the other PR with this change

**Pull request merged**